python/cudf/cudf/tests/test_binops.py

```diff
@@ -895,19 +909,11 @@ def test_binop_bool_uint(func, rhs):
 
 
 @pytest.mark.parametrize(
-    "series_dtype", (np.bool_, np.int8, np.uint8, np.int64, np.uint64)
+    "series_dtype", (np.int8, np.uint8, np.int64, np.uint64)
 )
 @pytest.mark.parametrize(
     "divisor_dtype",
     (
-        pytest.param(
-            np.bool_,
-            marks=pytest_xfail(
-                reason=(
-                    "Pandas handling of division by zero-bool is too strange"
-                )
-            ),
-        ),
```

Comment:
Moving the tests of bool operands into a separate test to xfail in a more fine-grained manner.

Reply:
Moved this into a separate branch.